dartsim/src/EntityManagementFeatures.hh

```diff
@@ -37,6 +37,7 @@ struct EntityManagementFeatureList : FeatureList<
   RemoveEntities,
   ConstructEmptyWorldFeature,
   ConstructEmptyModelFeature,
+  ConstructEmptyNestedModelFeature,
```

Comment:
Do we also need to add the `GetNestedModelFromModel` feature? It looks like methods from both the `ConstructEmptyNestedModelFeature` and `GetNestedModelFromModel` features are being implemented below.

Reply:
I believe that's already included in the `GetEntities` `FeatureList`. https://github.com/azeey/ign-physics/blob/dartsim_nested_entities/include/ignition/physics/GetEntities.hh#L510

Reply:
Thanks for pointing that out. Sorry for not seeing that the first time around!